src/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,30 @@
+package coffee.order;
+
+import java.util.*;
+
+public class CoffeeOrderBoard {
+
+    private TreeSet<Order> queue = new TreeSet<>();
+    private int queueNumber = 1;
+
+    public void add(String name) {
+        queue.add(new Order(name, queueNumber));
+        queueNumber++;
+    }
+
+    public void deliver() {
+        queue.pollFirst();
+    }
+
+    public void deliver(int queueNumber) {
+        queue.remove(new Order("", queueNumber));
+    }
+
+    public void draw() {
+        System.out.println("Num | name");
```

Comment:
How many times should I repeat that print statements must not be used in the methods? Make this method return a string of the structure you need and print it in the `main` method.